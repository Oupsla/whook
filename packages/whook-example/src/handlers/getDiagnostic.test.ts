import initGetDiagnostic from './getDiagnostic';

describe('getDiagnostic', () => {
  const TRANSACTIONS = {};

  it('should work', async () => {
    const getDiagnostic = await initGetDiagnostic({
      TRANSACTIONS,
    });
    const response = await getDiagnostic();

    expect({
      response,
    }).toMatchSnapshot();
  });
});
