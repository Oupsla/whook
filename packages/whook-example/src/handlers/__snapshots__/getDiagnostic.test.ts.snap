// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`getDiagnostic should work 1`] = `
Object {
  "response": Object {
    "body": Object {
      "transactions": Object {},
    },
    "status": 200,
  },
}
`;
